application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,28 @@
+;(
+    function () {
+        var errorBox = document.querySelector('.flash-message .alert-box.error');
+
+        if (!isUndefinedOrNull(errorBox)) {
+            var errors = errorBox.querySelectorAll('a')
```

Comment:
Rename to `errorLinks` just to be a bit more explicit? 

Possibly we should query for only links that jump somewhere else on the same page, just to protect against scenarios where this box might contain other links (although that seems unlikely)? Or we just add an explicit class to each of the links?